DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -57,119 +57,125 @@
+                    Log in
+                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                    </svg>
+                  </a>
+                </li>
+                <li class="nhsuk-header__navigation-item">
+                  <a class="nhsuk-header__navigation-link" asp-controller="Register" asp-action="Index">
+                    Register
+                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                    </svg>
+                  </a>
+                </li>
+              }
+              @if (User.Identity.IsAuthenticated) {
+                <li class="nhsuk-header__navigation-item">
+                  <a class="nhsuk-header__navigation-link" href="@Configuration["AppRootPath"]/MyAccount">
+                    My account
+                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                    </svg>
+                  </a>
+                </li>
+                <li class="nhsuk-header__navigation-item">
+                  <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/Home?action=appselect">
+                    Switch application
+                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                    </svg>
+                  </a>
+                </li>
+                <li class="nhsuk-header__navigation-item">
+                  <a class="nhsuk-header__navigation-link" href="@Configuration["CurrentSystemBaseUrl"]/logout">
+                    Log out
+                    <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                      <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                    </svg>
+                  </a>
+                </li>
+              }
+            </ul>
+          </div>
+        </nav>
+      }
+
+      @{ var visualSeparatorStyle = (bool?)ViewData["DoNotDisplayNavBar"] == true ? "display : block;" : string.Empty; }

```

Comment:
Could we add an extra class in the layout styles so the styling lives in the CSS files? :)